kconfigs/mediatek.conf

```diff
@@ -1 +1,4 @@
-CONFIG_DRM=y
+CONFIG_TEE=y
+CONFIG_OPTEE=y
+### This prevents random failures of "make check" in Travis CI
+CONFIG_PREEMPT=y
```

Comment:
You're right, it's there, I'll remove it, add the tags, and force push the update.